libvirt/tests/src/virtual_disks/virtual_disks_throttlefilters.py

```
@@ -0,0 +1,188 @@
+            disk_size = get_size_for_rbd(new_image_path, mon_host, server_user, server_pwd, test)
+        try:
+            return int(disk_size)
+        except (TypeError, ValueError):
+            test.fail("Failed to get image dsize for %s" % new_image_path)
+
+    def write_data(session):
+        """
+        Write data and discard data inside guest.
+        """
+        new_disk, _ = libvirt_disk.get_non_root_disk_name(session)
+        check_disk = session.cmd_output("lsblk -M").strip()
```

Ruff (0.14.10)

Local variable `check_disk` is assigned to but never used

Remove assignment to unused variable `check_disk`

(F841)